lib/newman_conway.py

```
@@ -6,5 +6,25 @@ def newman_conway(num):
     """ Returns a list of the Newman Conway numbers for the given value.
         Time Complexity: ?
         Space Complexity: ?
+
+        P(n) = P(P(n - 1)) + P(n - P(n - 1))
     """
-    pass
+    # base = ['1','1']
+    if num == 0:
+        raise ValueError()
```

We should raise this error for _any_ value below the valid starting point of the sequence:
```py
    if num <= 0:
        raise ValueError()
```